src/Framework/BinaryTranslator.cs

```diff
@@ -22,6 +22,14 @@ namespace Microsoft.Build.BackEnd
     /// </summary>
     internal static class BinaryTranslator
     {
+        /// <summary>
+        /// Presence of this key in the dictionary indicates that it was null.
+        /// </summary>
+        /// <remarks>
+        /// This is needed for a workaround.
```

Comment:
It would be nice to specify the workaround if possible.

Reply:
Added more information to comments.